plugins/modules/zos_ping.rexx

```diff
@@ -85,7 +85,7 @@ If (rc <> 0 | returnCode <> HWTJ_OK) Then Do
     failModule(errmsg, "", retC)
 End
 
-/* Check for Python version >= 3.8 eg: 'Python 3.8.2' */
+/* Check for Python version >= 3.8 eg: 'Python 3.10.0' */
```

Comment:
Trivial: Should the comment say version > 3.9, not >= 3.8?

Reply:
Good catch, its fixed now.